packages/lpa-questionnaire-web-app/src/app.js

```diff
@@ -21,7 +21,8 @@ const routes = require('./routes');
 const { VIEW } = require('./lib/views');
 
 try {
-	appInsights.setup().start();
+	appInsights.setup().setAutoDependencyCorrelation(true).setSendLiveMetrics(true).start();
+	appInsights.defaultClient.setAutoPopulateAzureProperties(true);
```

Comment:
I don't think this is actually deployed at present. I'd guess it's an older version of the LPA work we're currently planning.